src/UserProvider.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable react/prop-types */
```

Comment:
Don't disable rules in line, unless it's an exception, change the `eslint` config to what you like. Also search throughout your app and fix occurrences of this.